wagtailio/static/sass/components/_cta.scss

```
@@ -0,0 +1,46 @@
+.cta {
+    $root: &;
+    padding: 25px 30px;
+    text-decoration: none;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border: 1px solid $color--off-black;
+    border-radius: 10px;
+
+    > svg {
+        transition: transform 0.3s;
+        margin: 0 15px 0 auto;
+    }
+    
+    &:hover {
+        background-color: $color--indigo;
+        color: $color--white;
+
+        #{$root}__title {
+            color: $color--white;
+        }
+    
+        #{$root}__description {
+            color: $color--white;
+        }
```

I think this can be reduced to:
```suggestion
    &:hover {
        background-color: $color--indigo;

        #{$root}__title,
        #{$root}__description {
            color: $color--white;
        }
```

---

Good point!